test/e2e/feature/feature.go

```diff
@@ -392,6 +392,7 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	VolumeSnapshotDataSource = framework.WithFeature(framework.ValidFeatures.Add("VolumeSnapshotDataSource"))
 
+	VolumeGroupSnapshotDataSource = framework.WithFeature(framework.ValidFeatures.Add("volumegroupsnapshot"))
```

Comment:
please add a blank line after L395. Add a comment to indicate this feature is owned by sig-storage like L382

Reply:
Done